### PR TITLE
Added all changes commited to BuildingRobotics fork of python_saml

### PR DESCRIPTION
**Pull request by marianGlatzner**
Hope this has everything it needs. I wasn't sure about the travis file, since i've never handled those before.
Signed-off-by: Marian Glatzner <marian.glatzner@siemens.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -23,3 +23,26 @@ script:
...
+    secure: "Ipp50Lq+/x03czeUOpm7SP1Q+Zcn47LPrGgJsO2guwP1Y79ZvWw1ISgXQSRqkOO6I1OZVowS33J0qDda4UTn07EA4TnC73p1oG5RP7OTyz03yXXd20VJyXHvczfiXOrIDULylcRv2YQ4xojTh7opi6Lb3vhlLSgd4sPJJDqrkMeRiHf4K1s/8K8tbBFhPxXnXB+CCftgTC+f1nGZukfMcAmCzvnqCIqwZVrtXO3kwPTs40kKZWqMaDJSArbdOob21ydXuk+ajpC54Zkksl1hlJVKOASwLaDpIJSUOGpqB9iLwyqbtRy68PfZ7BMA/Ga0ZmLIOl2itVIZ0O7mrXem5Oz1t3DhApaa1D/YAuZISiK8sGm/m+nqim8XsbuISZclS4RMQliIwswuo1T1/kDdqxijBnHIq2vZhtba6qjaRYFgeg6pnoMYIjds9mpWOz0ilyAB5wQG9ZyJpjwELxEvMmsbo5PoeRWoXX2G3U9HeQWLb2fQY/o361eIWXrFUakYryOEdJvliTCRlUu7tR4i4FTAxhdLAPV51ngEJRfwlp4/cqT4fZHfNPYqp75JEb9pWfgG4e8r5sTrZxN+muucL+FWlVuF7r+SP6AE7ar8nFHQlwcxFLeUimVS3nj0Bjk/Rpt1aYOZ4mx/ihaUyDvGJqvrBRZ2vklT37YEeS/28gQ="
+  bucket: software-buildingrobotics
+  local-dir: wheel/dist
+  upload-dir: python_saml
```
Comment (jingo64) [inline .travis.yml]: Can you change this to python3_saml? Just to keep it separate from the existing python-saml code we have in there

File: .travis.yml [hunk 2]
```diff
@@ -23,3 +23,26 @@ script:
...
+  region: us-west-1
+  skip_cleanup: true
+  on:
+    repo: BuildingRobotics/python-saml
```
Comment (jingo64) [inline .travis.yml]: This should be python3-saml also

**Review by jingo64 — APPROVED** [review]
Hey! So this looks fine - honestly though if the only things we did in the PR were change some comments/refactor, I vote we just use their version straight up and save having this fork.

I recommend just using the vanilla python3-saml version inside the virtualenv instead of this fork, we can always come back and re-work the fork later if we think we need it, but otherwise we can just delete it.

**Comment by marianGlatzner** [comment]
> Hey! So this looks fine - honestly though if the only things we did in the PR were change some comments/refactor, I vote we just use their version straight up and save having this fork.
> 
> I recommend just using the vanilla python3-saml version inside the virtualenv instead of this fork, we can always come back and re-work the fork later if we think we need it, but otherwise we can just delete it.

Wouldn't it then be better to just take whichever version of python3-saml is most current, and putting that into our pypi repo? I mean if we're gonna use the vanilla version, without any additions specific to us, then this fork doesn't really serve a purpose anymore.

**Pull request merged**